Source/Controllers/SubviewControllers/SPTableInfo.m

```diff
@@ -196,8 +196,20 @@ - (void)tableChanged:(NSNotification *)notification
 				[info addObject:[NSString stringWithFormat:NSLocalizedString(@"size: %@", @"Table Info Section : table size on disk"), [NSString stringForByteSize:[[tableStatus objectForKey:@"Data_length"] longLongValue]]]];
 			}
 
+            if([[tableStatus objectForKey:@"Collation"] isNSNull]){
+                NSDictionary *userInfo = @{
+                    NSLocalizedDescriptionKey: @"tableStatus Collation is NSNull" ,
+                    @"serverVersion" : tableDocumentInstance.mySQLVersion,
+                };
+
+                [FIRCrashlytics.crashlytics recordError:[NSError errorWithDomain:@"database" code:4 userInfo:userInfo]];
+            }
+
+            SPLog(@"tableStatus: %@", tableStatus);
+            CLS_LOG(@"tableStatus: %@", tableStatus);
+
 			NSString *tableEnc = [tableDataInstance tableEncoding];
-			NSString *tableColl = [tableStatus objectForKey:@"Collation"];
```

Comment:
Do we want to define a fallback for if null? Like utf8mb4?

Reply:
It's just for the info dropdown ... not used for connection or anything.